english/mystory.md

```diff
@@ -0,0 +1,7 @@
+Once a cat was caught in a hunter's net. A mouse used to live in a nearby hole. The mouse seeing the cat in the net started playing around the cat. 
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.